modules/developer-cli-odo-listing-available-devfile-components.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// cli_reference/developer_cli_odo/creating-a-java-application-using-devfile
+
+[id="listing-available-devfile-components_{context}"]
+= Listing available devfile components
+
+:FeatureName: Listing available devfile components on a cluster in `{odo-title}`
+include::technology-preview.adoc[leveloffset=+1]
+
+With `{odo-title}`, you can display all the components that are available for you on the cluster. Note, that which components are available depends on the configuration of your cluster. 
+
+.Procedure
+
+. To list available devfile components on your cluster, run:
```

Comment:
For consistency with everything else, how about

> List available devfile components on your cluster:

I've been hamming away at "To [perform task], enter the following command", but not everyone follows that convention.